src/libmongoc/tests/unified/operation.c

```diff
@@ -1452,6 +1535,29 @@ operation_iterate_until_document_or_error (test_t *test,
    return ret;
 }
 
+static bool
+operation_close (test_t *test,
```

Comment:
This operation is just for cursor types. Should the function be named `operation_cursor_close` instead?

Edit: I just realized that isn't typically done for other operation methods, so this is fine as-is.

Reply:
Yeah, this surprised me as well, as the operations don't seem to be tied to objects. @kevinAlbs want me to create a ticket for this?

Reply:
The name of the operation functions should be 1:1 with the operation names from UTF, e.g. `operationFooBar` => `operation_foo_bar`.

I think the operation names must be unique in UTF. `operation_close` seems correct.

Reply:
> I think the operation names must be unique in UTF.

FWIW, this is not actually a requirement of the spec even if the names might appear unique in practice (if you're considering the variations of `aggregate` to be the same). I suppose we can punt on this until it actually becomes an issue.